idl/cherami_server/store.thrift

```diff
@@ -227,6 +227,19 @@ struct RemoteReplicateExtentRequest {
   3: optional string extentUUID
 }
 
+// ListExtentsElem contains information to identify an extent on the store
+// - destinationUUID: is the uuid of the destination, when available
+// - extentUUID: is the uuid of the extent
+struct ListExtentsElem {
+  1: optional string destinationUUID
+  2: optional string extentUUID
```

Comment:
great start, how about also adding other store extent metadata:

* ExtentType (AppendOnly|Timer)
* Created and/or modifiedTime
* Status (SEALED|OPEN)
* LastSequenceNumber
* BeginSequenceNumber
* SealSequenceNumber
* StartAddress
* EndAddress ?
* SizeInBytes